src/Core/Repositories/EntityFramework/BaseEntityFrameworkRepository.cs

```diff
@@ -18,5 +27,22 @@ public DatabaseContext GetDatabaseContext(IServiceScope serviceScope)
         {
             return serviceScope.ServiceProvider.GetRequiredService<DatabaseContext>();
         }
+
+        public void ClearChangeTracking()
+        {
+            using (var scope = ServiceScopeFactory.CreateScope())
+            {
+                var dbContext = GetDatabaseContext(scope);
```

Comment:
When clients hit the API the context is never reused. Following that flow, you're right that this method would never really accomplish anything.

`ClearChangeTracking()` is only ever used in the unit test project. There scope is scoped to a single test, instead of a single call. This was the best solution I could come up with at the time, but I couldn't reliably get Autofixture to supply the scope service on a per call basis. `ClearChangeTracking()` is used to supplement that and "refresh" the scope at appropriate times between repository calls.